src/asset-swapper/swap_quoter.ts

```diff
@@ -146,7 +146,11 @@ export class SwapQuoter {
      * Returns the recommended gas price for a fast transaction
      */
     public async getGasPriceEstimationOrThrowAsync(): Promise<BigNumber> {
-        return this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync();
+        const gasPrices = await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync({
```

Comment:
Here I think we shouldn't provide a default. (0.1 gwei as a sefault only makes sense on Arbitrum).